src/options.rs

```diff
@@ -164,6 +190,14 @@ impl Options {
         );
       }
     }
+
+    for descriptor in client.list_descriptors(None)?.descriptors {
+      let desc = descriptor.desc;
+      if !Regex::new(r"(tr|rawtr).*").unwrap().is_match(&desc) {
```

Comment:
This should be a little more specific, we should require exactly one tr descriptor, 0 or more rawtr descritpros, and no other descriptors.